index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hola Microverse</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <h3>¡¡Hola Microverse!!</h3>
+</html>
+</body>
```

Comment:
- Please correct the nesting of HTML tags for creating a better code according to [HTML best practices]().